src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -3,12 +3,14 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TELEGRAM;
 
-//import java.util.Set;
-import seedu.address.logic.commands.modulelistcommands.AddModuleCommand;
-import seedu.address.logic.commands.modulelistcommands.EditModuleDescriptor;
+import java.util.Set;
+
+import seedu.address.logic.commands.contactlistcommands.AddContactCommand;
+import seedu.address.logic.commands.contactlistcommands.EditContactDescriptor;
 import seedu.address.model.contact.Contact;
-//import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.Tag;
 
 /**
  * A utility class for Person.
```

Comment:
I think we can change `Person` to `Contact` if time allows.

Reply:
Thanks for the note, this was actually resolved in PR #598